### ISSUE TITLE
Default Container group not downloading collections/requirements.yml

### ISSUE THREAD

**Issue by felipe4334** [ansible/tower-operator#433]
I'm currently having an issue were when a job is created on a container group, and it triggers a job pod to be created. The new pod does not use the collections/requirements.yml file or even requirements.yml in the root directory of  the project. 

This is the error I get when running a job from a container group, it looks like "ansible collection location" does not get updated to represent the requirements.yml file.
![image](https://user-images.githubusercontent.com/13802820/123480393-4dcb1200-d5d0-11eb-8b84-6ebd16bab6b1.png) 

requirements.yml:
```
---
collections:
   - name: oracle.oci
```

project:
![image](https://user-images.githubusercontent.com/13802820/123481908-676d5900-d5d2-11eb-9dc6-a1217351f714.png)

When I write my own ansible.cfg and and download the collections and map a volume to the pods /etc/ansible location then everything works fine, but this solution is not scalable and will require for collections to manually be added every time a project is using newer collections. 
![image](https://user-images.githubusercontent.com/13802820/123481460-c088bd00-d5d1-11eb-8d98-d2af856f5b84.png)
```
apiVersion: v1
kind: Pod
metadata:
  namespace: awx
spec:
  dnsConfig:
    searches:
      - office.domain.com
      - prod.domain.com
  containers:
    - image: 'quay.io/ansible/awx-ee:0.4.0'
      name: worker
      args:
        - ansible-runner
        - worker
        - '--private-data-dir=/runner'
      volumeMounts:
        - name: oke-fsspvc-awx-etc-ansible
          mountPath: /etc/ansible
  volumes:
    - name: oke-fsspvc-awx-etc-ansible
      persistentVolumeClaim:
        claimName: oke-fsspvc-awx-etc-ansible
```

ansible.cfg
```
[defaults]
nocows = True
collections_paths = /etc/ansible/collections
roles_path = roles
library = /var/lib/awx/.ansible/plugins/modules:/usr/share/ansible/plugins/modules:/usr/share/ansible/collections/mdules:/etc/ansible/collections/ansible_collections/oracle/oci/plugins/modules
forks = 50

[ssh_connection]
pipelining = True
ssh_args = -o PreferredAuthentications=publickey
```

This issue appears to only happen with Container group and not with instance group. Although I cannot test the tower - instance group on my new Kubernetes awx operator deployment because I get a "Status: pending: This job is not ready to start because there is not enough available capacity." error, even though my awx_tasks requirements are fairly high `Request CPU 10000m and memory 15Gi` when testing on my old awx deployment when the project runs out of awx_tasks the collections get automatically downloaded.

This issue is happening with both customized or non-customized pod specification.

**Comment by felipe4334** [comment]
Any help would be appreciated please :)

**Comment by metabsd** [comment]
You have to review what's coming with awx-ee:0.4.0 and maybe fork it and add the dependencies you need for that specific collection.

**Comment by felipe4334** [comment]
I ended up building a custom image of awx-ee:0.4.0 with the dependencies I needed ex. "ansible-galaxy collection requirements.yml" then I specified the custom image on the container group and the execution environment. Make sure your playbooks have the dependencies specified on the correct file and not on the playbook.yml. If you have for example community.windows on your projects dependencies then you only need to use the module names for example copy_file on your projects, not community.windows.copy_file
